src/main/java/com/codegile/univ/mihai/week1/day1/Project3a.java

```diff
@@ -0,0 +1,57 @@
+package com.codegile.univ.mihai.week1.day1;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Collections;
+
+
+public class Project3a {
+    public static void main(String args[]){
+        FileInputStream fis;
+        InputStreamReader isr;
+        BufferedReader br;
+
+        try{
+            fis = new FileInputStream("src/com/company/word.txt");
+            isr = new InputStreamReader(fis);
+            br = new BufferedReader(isr);
+            ArrayList<String> obj = new ArrayList<String>();
```

Comment:
try to use interface declaration, and implementation only as the value:
`List<String> obj = new ArrayList<String>();`
try to name variables according to their purpose:
`List<String> words = new ArrayList<String>();`
in newer versions of Java you can also get rid of the type declaration on the right-hand side, read about Type Inference:
`List<String> words = new ArrayList<>();`